skeleton/default.nix

```diff
@@ -1,16 +1,17 @@
 { obelisk ? import ./.obelisk/impl {
-  system = builtins.currentSystem;
-  iosSdkVersion = "10.2";
-  # You must accept the Android Software Development Kit License Agreement at
-  # https://developer.android.com/studio/terms in order to build Android apps.
-  # Uncomment and set this to `true` to indicate your acceptance:
-  # config.android_sdk.accept_license = false;
+    system = builtins.currentSystem;
+    iosSdkVersion = "10.2";
+    # You must accept the Android Software Development Kit License Agreement at
+    # https://developer.android.com/studio/terms in order to build Android apps.
+    # Uncomment and set this to `true` to indicate your acceptance:
+    # config.android_sdk.accept_license = false;
   }
+, projectOverrides ? {}
```

Comment:
Why encode this as part of the skeleton? Do you see this being used by the average obelisk user? It's easy enough to do yourself you really want it...